### PR TITLE
Update and restructure BTP Manager docs

### PR DESCRIPTION
**Pull request by IwonaLanger**
**Description**

Changes proposed in this pull request:

- Update the main `README.md` file
- Add the `installation.md` file
- minor language corrections
- minor formatting corrections

**Related issue(s)**
See [#16421](https://github.com/kyma-project/kyma/issues/16421)

### CODE REVIEW

File: docs/operations.md [hunk 1]
```diff
@@ -93,7 +93,9 @@ In order to delete finalizers the reconciler deletes module deployment and webho
 Regardless of mode, in the next step, all SAP BTP Service Operator resources marked with the `app.kubernetes.io/managed-by:btp-manager`
 label are deleted. The deletion process of module resources is based on resources GVKs (GroupVersionKinds) found in [manifests](../module-resources).
 If the process succeeds, the finalizer on BtpOperator CR itself is removed and the resource is deleted.
-If an error occurs during the deprovisioning, state of BtpOperator CR is set to `Error`.
+If an error occurs during the deprovisioning, state of BtpOperator CR is set to `Error`. 
+
+The deprovisioning process is presented in the following diagram:
```
Comment (grego952) [inline docs/operations.md:98]:
```suggestion
You can see the deprovisioning process in the following diagram:
```

File: docs/operations.md [hunk 2]
```diff
@@ -69,6 +65,10 @@ set of [timeouts](../controllers/btpoperator_controller.go) defined as `consts`
 for performed operations. The provisioning is successful when all module resources exist in the cluster. This is the
 condition which allows the reconciler to set the CR in `Ready` state.
 
+The provisioning process is carried out as presented in the following diagram:
```
Comment (grego952) [inline docs/operations.md:68]:
```suggestion
You can see the provisioning process in the following diagram:
```

File: README.md [hunk 3]
```diff
@@ -70,6 +23,10 @@ kubectl apply -f deployments/prerequisites.yaml
 kubectl apply -f examples/btp-manager-secret.yaml
 kubectl apply -f examples/btp-operator.yaml
 ```
+To create the BTP Manager Secret, follow these steps:  
+1. create a service binding to obtain the access credentials to the service instance as described in points 2b and 2c of the [Setup](https://github.com/SAP/sap-btp-service-operator#setup) section in the SAP BTP Service Operator documentation
```
[the remark this answers is not in the document]
Comment (IwonaLanger) [reply README.md:27]: @grego952 It surely is. Thank you for spotting it. Looking [here](https://help.sap.com/docs/service-manager/sap-service-manager/creating-service-bindings-in-other-environments?locale=en-US) and [here](https://github.com/SAP/sap-btp-service-operator) was a bit confounding, though :)

**Review by grego952 — DISMISSED** [review]

File: docs/installation.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+---
+
+## Install BTP Manager locally
+
+### Prerequisites
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- Kubernetes cluster (you can use [k3d](https://k3d.io)) 
+
+Use the following commands to run the BTP Manager controller from your host. Both `make` commands refer to [Makefile](../Makefile) in the `operator` directory.
```
Comment (szwedm) [inline docs/installation.md:12]: `operator` dir doesn't exist anymore

Comment (IwonaLanger) [reply docs/installation.md]: @szwedm Is the Makefile okay?

Comment (szwedm) [reply docs/installation.md]: @IwonaLanger yes, referenced Makefile is correct.

Comment (IwonaLanger) [reply docs/installation.md]: Thank you @szwedm

File: README.md [hunk 5]
```diff
@@ -70,6 +23,10 @@ kubectl apply -f deployments/prerequisites.yaml
 kubectl apply -f examples/btp-manager-secret.yaml
```
[the remark this answers is not in the document]
Comment (PK85) [reply README.md:23]: whats about that?

**Review by PK85 — APPROVED** [review]

File: docs/operations.md [hunk 6]
```diff
@@ -21,16 +21,12 @@ The prerequisites for SAP BTP Service Operator provisioning are:
...
 the prerequisites yourself.
 
 ### Process
 
-The provisioning process is part of a module reconciliation and is carried out as presented in the following diagram:
-
-![Provisioning diagram](./assets/provisioning.svg)
-
-Create a [BtpOperator CR](../api/v1alpha1/btpoperator_types.go) to trigger the reconciliation:
+The provisioning process is part of a module reconciliation. To trigger the reconciliation, create a [BtpOperator CR](../api/v1alpha1/btpoperator_types.go) :
```
Comment (grego952) [inline docs/operations.md:29]:
```suggestion
The provisioning process is part of a module reconciliation. To trigger the reconciliation, create a [BtpOperator CR](../api/v1alpha1/btpoperator_types.go):
```

**Pull request merged**